packages/angular/projects/dev/tsconfig.app.json

```diff
@@ -5,5 +5,14 @@
     "types": []
   },
   "files": ["src/main.ts", "src/polyfills.ts"],
-  "include": ["src/**/*.d.ts"]
+  "include": ["src/**/*.d.ts"],
+  "exclude": ["**/*.stories.ts"],
+  "paths": {
+    "@clr/core": ["./dist/core"],
```

Comment:
These should be updated at the packages/angular/tsconfig.json file instead.

Reply:
Confirmed and removed.